internal/e2e/full_suit_test.go

```diff
@@ -72,8 +72,13 @@ func Test(t *testing.T) {
 					New: func() *cobra.Command {
 						return cmd.NewRootCmd(nil)
 					},
-					Ctx:            ctx,
-					PersistentArgs: []string{"--" + cliclient.FlagReadRemote, reg.Config(ctx).ReadAPIListenOn(), "--" + cliclient.FlagWriteRemote, reg.Config(ctx).WriteAPIListenOn(), "--" + cmdx.FlagFormat, string(cmdx.FormatJSON)},
+					Ctx: ctx,
+					PersistentArgs: []string{
+						"--" + cliclient.FlagReadRemote, reg.Config(ctx).ReadAPIListenOn(),
+						"--" + cliclient.FlagWriteRemote, reg.Config(ctx).WriteAPIListenOn(),
+						"--insecure-disable-transport-security=true",
```

Comment:
Maybe we can run this e2e test using the self-signed cert?

Reply:
Then we'd have to solve the multiplexing issue (which I think is a new feature).

Reply:
Then we'd have to solve the multiplexing issue (which I think is a new feature).

Reply:
Then we'd have to solve the multiplexing issue (which I think is a new feature).